addons/sourcemod/scripting/soap_tf2dm.sp

```diff
@@ -2082,3 +2098,32 @@ public OnPluginEnd()
 {
     MC_PrintToChatAll(SOAP_TAG ... "Soap DM unloaded.");
 }
+
+public bool GetConfigPath(const char[] map, char[] path, int maxlength)
+{
+    LogMessage("No config for: %s, searching for fallback", map);
+    char cleanMap[64];
+    strcopy(cleanMap, sizeof(cleanMap), map);
+    Regex normalizeMapRegex = new Regex("(_(a|b|beta|u|r|v|rc|f|final|comptf|ugc)?[0-9]*[a-z]?$)|([0-9]+[a-z]?$)", 0);
```

Comment:
Pretty sure this needs to be added as a global `Regex normalizeMapRegex` and then init'd in OnPluginStart `normalizeMapRegex = new Regex("(_(a|b|beta|u|r|v|rc|f|final|comptf|ugc)?[0-9]*[a-z]?$)|([0-9]+[a-z]?$)", 0);` instead reinit'd every time this func is called